src/App.css

```diff
@@ -0,0 +1,42 @@
+#root {
```

Comment:
are these css being used anywhere? If not then please remove them from the commit.

Reply:
You can keep the files just remove the content inside because these files are required for future usecases